tests/testthat/test-list_distributions.R

```diff
@@ -119,19 +122,56 @@ test_that("list_distributions works for serial_interval, params = TRUE", {
     parameters = TRUE
   )
   expect_true(is.data.frame(serial_inter_tbl))
-  expect_equal(dim(serial_inter_tbl), c(3, 22))
+  expect_equal(dim(serial_inter_tbl), c(5, 27))
```

Comment:
Broader question - should we try and avoid hard-coded values in the checks? E.g. by referring to original data file dimensions?

Reply:
These check are slightly redundant as we have a snapshot test (`expect_snapshot`) which checks against the last recorded version of the data frame. I just added these as I wasn't familiar with snapshots and wanted to make sure they worked as expected by writing some extra checks. These can be deleted if the size of the database frequently changes.